apps/tx-processor/src/auth.ts

```diff
@@ -0,0 +1,41 @@
+import http from 'http';
+import url from 'url';
+import { log } from '@thecointech/logging';
+import open from "open";
+
+export function getCode(url: string) {
+  log.debug('Begin OAuth process');
+  open(url);
+  return listenForCode();
+}
+
+// Simple callback to wait for redirect from Google
+function listenForCode() {
```

Comment:
Nicer version of authentication.

At some point, we'll probably want to generalize this so we can use OAuth login for interactions with the DB as well (and ditch the service accounts we currently use).